lava_test_plans/projects/lkft-android/devices/variables/hi6220-hikey-r2.yaml

```diff
@@ -0,0 +1,8 @@
+DEVICE_TYPE: hi6220-hikey
+KERNEL_URL: unknown
+DTB_URL: unknown
+BOOT_URL: unknown
+MODULES_URL: unknown
+ROOTFS_URL: unknown
+LAVA_JOB_PRIORITY: 25
+PROJECT: projects/lkft/
```

Comment:
should this be 'projects/lkft-android/' ?

Reply:
done.